### DISCUSSION TITLE
Vim shortcuts like autocorrect aa => @

### DISCUSSION THREAD

**Discussion by hmt** [zed-industries/zed#6709] [Feature Requests]
I have mapped many of my keys on my German keyboard to quickly get hard to type characters like `@, [, ], {, }` via `aa`, `öö`, `ää` etc.
Is this possible in zed?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
